BitwardenShared/UI/Platform/Application/Appearance/UI.swift

```diff
@@ -87,10 +87,14 @@ public enum UI {
         tabBarAppearance.inlineLayoutAppearance.normal.badgeTextAttributes = iconBadgeTextAttributes
         tabBarAppearance.stackedLayoutAppearance.normal.badgeBackgroundColor = iconBadgeBackground
         tabBarAppearance.stackedLayoutAppearance.normal.badgeTextAttributes = iconBadgeTextAttributes
+        tabBarAppearance.stackedLayoutAppearance.selected.titleTextAttributes = [
+            .foregroundColor: Asset.Colors.iconSecondary.color,
+        ]
+        tabBarAppearance.stackedLayoutAppearance.normal.titleTextAttributes = [
+            .foregroundColor: Asset.Colors.iconPrimary.color,
+        ]
```

Comment:
@matt-livefront realized the text color wasn't being changed correctly and after some research from dr google it looks like we needed to use this instead.

Reply:
Thanks! I think we also need to do this for the `inlineLayoutAppearance` and `stackedLayoutAppearance` above.